ac_dc/oscar_sample_filter.py

```diff
@@ -105,13 +141,33 @@ def remove_long_words(
         filtered_sentence = " ".join(words)
         return filtered_sentence
 
+    @staticmethod
+    def apply_regex_govt_id_anonymization(sentence, lang_id):
+        for regex in list(
+            ModifyingSentences.govt_id_regex.get(
+                lang_id, ModifyingSentences.govt_id_regex["any"]
+            ).values()
+        ):
+            matched = False
+            for ent in regex.findall(sentence):
+                if type(ent) != str:
```

Comment:
i just being nitpicky :) 

```suggestion
                if not isinstance(en, str):
```

Reply:
This looks great. But how do I accept your changes :) I hate github...